clang/test/Driver/sycl-offload-header-check.cpp

```diff
@@ -0,0 +1,11 @@
+// REQUIRES: system-windows
+// Test the integrated header file name generation.  Name should match the
```

Comment:
```suggestion
// Test the integration header file name generation.  Name should match the
```

Reply:
+1. I think amending the commit message might be necessary as well.